git-multimail/README

```diff
@@ -388,6 +394,18 @@ multimailhook.replyToRefchange
     - The value "none", in which case the Reply-To: field will be
       omitted.
 
+multimailhook.refFilterInclusionRegex
+multimailhook.refFilterExclusionRegex
+
+    Regular expressions that can be used to limit refs for which email
+    updates will be sent.  It is an error to specify both an inclusion
+    and an exclusion regex.  If an inclusion regex is specified, emails
```

Comment:
I'm wondering whether we should accept both options to be set, possibly with a third option to specify the order (like Apache's `Order Deny,Allow`). Or maybe this would be overkill.